clusterloader2/testing/load/experimental-config.yaml

```diff
@@ -424,6 +424,7 @@ steps:
     Params:
       action: start
       labelSelector: group = scheduler-throughput
+      measurmentInterval: 1s
```

Comment:
This is great, if we make this change then we can also reduce the number of the "scheduler-throughput" pods. We don't need 1000 anymore, something like 300 should be enough.

Reply:
This is only experimental config - if this will work, I will work on lowering those numbers. Right now it's only a matter of few seconds. Please keep it as is.

Reply:
Personally, I'd just do these two steps at once, but you're the owner of this issue, it's your call :)